cloudos_cli/clos.py

```diff
@@ -462,6 +464,166 @@ def process_job_list(r, all_fields=False):
+                            duration = end_dt - start_dt
+                            
+                            # Format duration as hours:minutes:seconds
+                            total_seconds = int(duration.total_seconds())
+                            hours = total_seconds // 3600
+                            minutes = (total_seconds % 3600) // 60
+                            seconds = total_seconds % 60
+                            
+                            if hours > 0:
+                                return f"{hours}h {minutes}m {seconds}s"
+                            elif minutes > 0:
+                                return f"{minutes}m {seconds}s"
+                            else:
+                                return f"{seconds}s"
+                        except (ValueError, TypeError):
+                            return None
+                return None
+
+            jobs_df['Run time'] = jobs_df.apply(calculate_runtime, axis=1)
+
+        # 5. Format batch.enabled (True -> "Batch", else "N/A")
+        if 'batch.enabled' in jobs_df.columns:
+            jobs_df['batch.enabled'] = jobs_df['batch.enabled'].apply(
+                lambda x: "Batch" if x is True else "N/A"
+            )
+
+        # 6. Rename columns using the provided dictionary
+        column_name_mapping = {
+            "status": "Status",
+            "name": "Name",
+            "parameters": "Parameters",
+            "project.name": "Project",
+            "user": "Owner",
+            "workflow.name": "Pipeline",
+            "createdAt": "Submit time",
+            "updatedAt": "End time", 
+            "revision.commit": "Commit",
+            "realInstancesExecutionCost": "Cost",
+            "masterInstance.usedInstance.type": "Resources",
+            "storageMode": "Storage type",
+            "workflow.repository.url": "Pipeline url",
+            "nextflowVersion": "Nextflow version",
+            "batch.enabled": "Executor",
+            "storageSizeInGb": "Storage size",
+            "batch.jobQueue.id": "Job queue",
+            "usesFusionFileSystem": "Accelerated file staging"
+        }
+
+        # Rename columns that exist in the DataFrame
+        jobs_df = jobs_df.rename(columns=column_name_mapping)
+
+        # Remove the original startTime and endTime columns since we now have Submit time, End time, and Run time
+        jobs_df = jobs_df.drop(columns=['startTime', 'endTime'], errors='ignore')
+
+        # Remove the _id column if it exists
+        if '_id' in jobs_df.columns:
+            jobs_df = jobs_df.drop(columns=['_id'])
+
+        # 7. Define the desired order of columns
+        desired_order = [
+            "Status", "Name", "Project", "Owner", "Pipeline", 
+            "Submit time", "End time", "Run time", "Commit", "Cost", 
+            "Resources", "Storage type", "Pipeline url", "Parameters", 
+            "Nextflow version", "Executor", "Storage size", "Job queue", 
```

Comment:
```suggestion
            "Status", "Name", "Project", "Owner", "Pipeline",
            "Submit time", "End time", "Run time", "Commit", "Cost",
            "Resources", "Storage type", "Pipeline url", "Parameters",
            "Nextflow version", "Executor", "Storage size", "Job queue",
```
trailing whitespaces